<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="49" passed="33" failed="14" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-09-20T01:54:08 ICT" name="Suite" finished-at="2022-09-20T02:03:42 ICT" duration-ms="573450">
    <groups>
    </groups>
    <test started-at="2022-09-20T01:54:08 ICT" name="Test1" finished-at="2022-09-20T02:03:42 ICT" duration-ms="573450">
      <class name="TestSuite.Logout">
        <test-method is-config="true" signature="beforeClass(java.lang.String)[pri:0, instance:TestSuite.Logout@4313f5bc]" started-at="2022-09-20T02:03:38 ICT" name="beforeClass" finished-at="2022-09-20T02:03:38 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.Logout@4313f5bc]" started-at="2022-09-20T02:03:38 ICT" name="setUpMethod" finished-at="2022-09-20T02:03:41 ICT" duration-ms="3345" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.Logout.MODULE3_01() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE3_01()[pri:0, instance:TestSuite.Logout@4313f5bc]" started-at="2022-09-20T02:03:41 ICT" name="MODULE3_01" finished-at="2022-09-20T02:03:42 ICT" duration-ms="503" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE3_01 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.Logout@4313f5bc]" started-at="2022-09-20T02:03:42 ICT" name="tearDownMethod" finished-at="2022-09-20T02:03:42 ICT" duration-ms="73" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE3_01 status=SUCCESS method=Logout.MODULE3_01()[pri:0, instance:TestSuite.Logout@4313f5bc] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.Logout.MODULE3_01() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.Logout@4313f5bc]" started-at="2022-09-20T02:03:42 ICT" name="afterMethod" finished-at="2022-09-20T02:03:42 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE3_01 status=SUCCESS method=Logout.MODULE3_01()[pri:0, instance:TestSuite.Logout@4313f5bc] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.Logout.MODULE3_01() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- TestSuite.Logout -->
      <class name="TestSuite.Login">
        <test-method is-config="true" signature="beforeClass(java.lang.String)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:21 ICT" name="beforeClass" finished-at="2022-09-20T02:03:21 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:21 ICT" name="setUpMethod" finished-at="2022-09-20T02:03:22 ICT" duration-ms="1623" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_01() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE1_01()[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:22 ICT" name="MODULE1_01" finished-at="2022-09-20T02:03:24 ICT" duration-ms="2132" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE1_01 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:24 ICT" name="tearDownMethod" finished-at="2022-09-20T02:03:25 ICT" duration-ms="80" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE1_01 status=SUCCESS method=Login.MODULE1_01()[pri:0, instance:TestSuite.Login@161479c6] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_01() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:25 ICT" name="afterMethod" finished-at="2022-09-20T02:03:25 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE1_01 status=SUCCESS method=Login.MODULE1_01()[pri:0, instance:TestSuite.Login@161479c6] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_01() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:25 ICT" name="setUpMethod" finished-at="2022-09-20T02:03:26 ICT" duration-ms="1669" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_02() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE1_02()[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:26 ICT" name="MODULE1_02" finished-at="2022-09-20T02:03:27 ICT" duration-ms="589" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE1_02 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:27 ICT" name="tearDownMethod" finished-at="2022-09-20T02:03:27 ICT" duration-ms="85" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE1_02 status=SUCCESS method=Login.MODULE1_02()[pri:0, instance:TestSuite.Login@161479c6] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_02() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:27 ICT" name="afterMethod" finished-at="2022-09-20T02:03:27 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE1_02 status=SUCCESS method=Login.MODULE1_02()[pri:0, instance:TestSuite.Login@161479c6] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_02() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:27 ICT" name="setUpMethod" finished-at="2022-09-20T02:03:28 ICT" duration-ms="1597" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_03() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE1_03()[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:28 ICT" name="MODULE1_03" finished-at="2022-09-20T02:03:29 ICT" duration-ms="605" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE1_03 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:29 ICT" name="tearDownMethod" finished-at="2022-09-20T02:03:29 ICT" duration-ms="73" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE1_03 status=SUCCESS method=Login.MODULE1_03()[pri:0, instance:TestSuite.Login@161479c6] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_03() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:29 ICT" name="afterMethod" finished-at="2022-09-20T02:03:29 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE1_03 status=SUCCESS method=Login.MODULE1_03()[pri:0, instance:TestSuite.Login@161479c6] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_03() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:29 ICT" name="setUpMethod" finished-at="2022-09-20T02:03:31 ICT" duration-ms="1545" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_04() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE1_04()[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:31 ICT" name="MODULE1_04" finished-at="2022-09-20T02:03:32 ICT" duration-ms="1231" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE1_04 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:32 ICT" name="tearDownMethod" finished-at="2022-09-20T02:03:32 ICT" duration-ms="75" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE1_04 status=SUCCESS method=Login.MODULE1_04()[pri:0, instance:TestSuite.Login@161479c6] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_04() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:32 ICT" name="afterMethod" finished-at="2022-09-20T02:03:32 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE1_04 status=SUCCESS method=Login.MODULE1_04()[pri:0, instance:TestSuite.Login@161479c6] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_04() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:32 ICT" name="setUpMethod" finished-at="2022-09-20T02:03:34 ICT" duration-ms="1527" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_05() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE1_05()[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:34 ICT" name="MODULE1_05" finished-at="2022-09-20T02:03:35 ICT" duration-ms="1258" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE1_05 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:35 ICT" name="tearDownMethod" finished-at="2022-09-20T02:03:35 ICT" duration-ms="72" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE1_05 status=SUCCESS method=Login.MODULE1_05()[pri:0, instance:TestSuite.Login@161479c6] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_05() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:35 ICT" name="afterMethod" finished-at="2022-09-20T02:03:35 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE1_05 status=SUCCESS method=Login.MODULE1_05()[pri:0, instance:TestSuite.Login@161479c6] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_05() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:35 ICT" name="setUpMethod" finished-at="2022-09-20T02:03:36 ICT" duration-ms="1569" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_06() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE1_06()[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:36 ICT" name="MODULE1_06" finished-at="2022-09-20T02:03:38 ICT" duration-ms="1272" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE1_06 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:38 ICT" name="tearDownMethod" finished-at="2022-09-20T02:03:38 ICT" duration-ms="75" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE1_06 status=SUCCESS method=Login.MODULE1_06()[pri:0, instance:TestSuite.Login@161479c6] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_06() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.Login@161479c6]" started-at="2022-09-20T02:03:38 ICT" name="afterMethod" finished-at="2022-09-20T02:03:38 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE1_06 status=SUCCESS method=Login.MODULE1_06()[pri:0, instance:TestSuite.Login@161479c6] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.Login.MODULE1_06() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- TestSuite.Login -->
      <class name="TestSuite.EnterPlan">
        <test-method is-config="true" signature="beforeSuite()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:54:08 ICT" name="beforeSuite" finished-at="2022-09-20T01:54:08 ICT" duration-ms="71" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method is-config="true" signature="beforeClass(java.lang.String)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:54:08 ICT" name="beforeClass" finished-at="2022-09-20T01:54:08 ICT" duration-ms="5" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:54:08 ICT" name="setUpMethod" finished-at="2022-09-20T01:54:15 ICT" duration-ms="6446" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_01() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_01()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:54:15 ICT" name="MODULE2_01" finished-at="2022-09-20T01:54:24 ICT" duration-ms="8822" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_01 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:54:24 ICT" name="tearDownMethod" finished-at="2022-09-20T01:54:24 ICT" duration-ms="81" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_01 status=SUCCESS method=EnterPlan.MODULE2_01()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_01() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:54:24 ICT" name="afterMethod" finished-at="2022-09-20T01:54:24 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_01 status=SUCCESS method=EnterPlan.MODULE2_01()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_01() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:54:24 ICT" name="setUpMethod" finished-at="2022-09-20T01:54:29 ICT" duration-ms="5222" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_02() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_02()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:54:29 ICT" name="MODULE2_02" finished-at="2022-09-20T01:54:41 ICT" duration-ms="11643" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_02 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:54:41 ICT" name="tearDownMethod" finished-at="2022-09-20T01:54:41 ICT" duration-ms="79" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_02 status=SUCCESS method=EnterPlan.MODULE2_02()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_02() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:54:41 ICT" name="setUpMethod" finished-at="2022-09-20T01:54:46 ICT" duration-ms="5739" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_03() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:54:41 ICT" name="afterMethod" finished-at="2022-09-20T01:54:41 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_02 status=SUCCESS method=EnterPlan.MODULE2_02()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_02() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method signature="MODULE2_03()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:54:46 ICT" name="MODULE2_03" finished-at="2022-09-20T01:54:55 ICT" duration-ms="8924" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_03 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:54:55 ICT" name="tearDownMethod" finished-at="2022-09-20T01:54:55 ICT" duration-ms="77" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_03 status=SUCCESS method=EnterPlan.MODULE2_03()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_03() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:54:55 ICT" name="afterMethod" finished-at="2022-09-20T01:54:55 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_03 status=SUCCESS method=EnterPlan.MODULE2_03()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_03() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:54:55 ICT" name="setUpMethod" finished-at="2022-09-20T01:55:01 ICT" duration-ms="5384" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_04() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_04()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:55:01 ICT" name="MODULE2_04" finished-at="2022-09-20T01:55:19 ICT" duration-ms="18385" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_04 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:55:19 ICT" name="tearDownMethod" finished-at="2022-09-20T01:55:19 ICT" duration-ms="89" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_04 status=SUCCESS method=EnterPlan.MODULE2_04()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_04() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:55:19 ICT" name="afterMethod" finished-at="2022-09-20T01:55:19 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_04 status=SUCCESS method=EnterPlan.MODULE2_04()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_04() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:55:19 ICT" name="setUpMethod" finished-at="2022-09-20T01:55:25 ICT" duration-ms="5488" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_05() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_05()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:55:25 ICT" name="MODULE2_05" finished-at="2022-09-20T01:55:34 ICT" duration-ms="9365" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_05 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:55:34 ICT" name="tearDownMethod" finished-at="2022-09-20T01:55:34 ICT" duration-ms="79" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_05 status=SUCCESS method=EnterPlan.MODULE2_05()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_05() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:55:34 ICT" name="afterMethod" finished-at="2022-09-20T01:55:34 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_05 status=SUCCESS method=EnterPlan.MODULE2_05()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_05() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:55:34 ICT" name="setUpMethod" finished-at="2022-09-20T01:55:39 ICT" duration-ms="5162" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_06() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_06()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:55:39 ICT" name="MODULE2_06" finished-at="2022-09-20T01:55:51 ICT" duration-ms="11887" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_06 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:55:51 ICT" name="tearDownMethod" finished-at="2022-09-20T01:55:51 ICT" duration-ms="84" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_06 status=SUCCESS method=EnterPlan.MODULE2_06()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_06() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:55:51 ICT" name="setUpMethod" finished-at="2022-09-20T01:55:57 ICT" duration-ms="5295" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_07() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:55:51 ICT" name="afterMethod" finished-at="2022-09-20T01:55:51 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_06 status=SUCCESS method=EnterPlan.MODULE2_06()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_06() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method signature="MODULE2_07()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:55:57 ICT" name="MODULE2_07" finished-at="2022-09-20T01:56:08 ICT" duration-ms="11762" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_07 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:08 ICT" name="tearDownMethod" finished-at="2022-09-20T01:56:08 ICT" duration-ms="85" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_07 status=SUCCESS method=EnterPlan.MODULE2_07()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_07() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:08 ICT" name="setUpMethod" finished-at="2022-09-20T01:56:14 ICT" duration-ms="5165" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_08() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:08 ICT" name="afterMethod" finished-at="2022-09-20T01:56:08 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_07 status=SUCCESS method=EnterPlan.MODULE2_07()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_07() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method signature="MODULE2_08()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:14 ICT" name="MODULE2_08" finished-at="2022-09-20T01:56:14 ICT" duration-ms="521" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Element with locator By.xpath: //p[contains(text(),'Ngày không được phép nhỏ hơn giá trị nhỏ nhất')] doesn't exist]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Element with locator By.xpath: //p[contains(text(),'Ngày không được phép nhỏ hơn giá trị nhỏ nhất')] doesn't exist
at org.testng.Assert.fail(Assert.java:97)
at Common.Utilities.assertElementVisible(Utilities.java:585)
at CommonScreen.EnterPlanScreen.checkDate(EnterPlanScreen.java:138)
at TestSuite.EnterPlan.MODULE2_08(EnterPlan.java:71)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:687)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:140)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at org.testng.TestRunner.privateRun(TestRunner.java:724)
at org.testng.TestRunner.run(TestRunner.java:574)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)
at org.testng.SuiteRunner.run(SuiteRunner.java:302)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1142)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1064)
at org.testng.TestNG.runSuites(TestNG.java:994)
at org.testng.TestNG.run(TestNG.java:962)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_08 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:14 ICT" name="tearDownMethod" finished-at="2022-09-20T01:56:14 ICT" duration-ms="271" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_08 status=FAILURE method=EnterPlan.MODULE2_08()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_08() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:14 ICT" name="afterMethod" finished-at="2022-09-20T01:56:14 ICT" duration-ms="6" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_08 status=FAILURE method=EnterPlan.MODULE2_08()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_08() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:14 ICT" name="setUpMethod" finished-at="2022-09-20T01:56:19 ICT" duration-ms="4411" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_09() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_09()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:19 ICT" name="MODULE2_09" finished-at="2022-09-20T01:56:20 ICT" duration-ms="826" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Element with locator By.xpath: //p[contains(text(),'Ngày không được phép lớn hơn giá trị lớn nhất')] doesn't exist]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Element with locator By.xpath: //p[contains(text(),'Ngày không được phép lớn hơn giá trị lớn nhất')] doesn't exist
at org.testng.Assert.fail(Assert.java:97)
at Common.Utilities.assertElementVisible(Utilities.java:585)
at CommonScreen.EnterPlanScreen.checkDate(EnterPlanScreen.java:138)
at TestSuite.EnterPlan.MODULE2_09(EnterPlan.java:76)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:687)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:140)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at org.testng.TestRunner.privateRun(TestRunner.java:724)
at org.testng.TestRunner.run(TestRunner.java:574)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)
at org.testng.SuiteRunner.run(SuiteRunner.java:302)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1142)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1064)
at org.testng.TestNG.runSuites(TestNG.java:994)
at org.testng.TestNG.run(TestNG.java:962)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_09 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:20 ICT" name="tearDownMethod" finished-at="2022-09-20T01:56:20 ICT" duration-ms="267" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_09 status=FAILURE method=EnterPlan.MODULE2_09()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_09() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:20 ICT" name="afterMethod" finished-at="2022-09-20T01:56:20 ICT" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_09 status=FAILURE method=EnterPlan.MODULE2_09()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_09() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:20 ICT" name="setUpMethod" finished-at="2022-09-20T01:56:25 ICT" duration-ms="5202" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_10() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_10()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:25 ICT" name="MODULE2_10" finished-at="2022-09-20T01:56:37 ICT" duration-ms="11653" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_10 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:37 ICT" name="tearDownMethod" finished-at="2022-09-20T01:56:37 ICT" duration-ms="77" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_10 status=SUCCESS method=EnterPlan.MODULE2_10()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_10() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:37 ICT" name="afterMethod" finished-at="2022-09-20T01:56:37 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_10 status=SUCCESS method=EnterPlan.MODULE2_10()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_10() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:37 ICT" name="setUpMethod" finished-at="2022-09-20T01:56:42 ICT" duration-ms="5137" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_11() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_11()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:42 ICT" name="MODULE2_11" finished-at="2022-09-20T01:56:43 ICT" duration-ms="1196" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Element with locator By.xpath: //p[contains(text(),'Ngày kết thúc không được trước ngày bắt đầu')] doesn't exist]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Element with locator By.xpath: //p[contains(text(),'Ngày kết thúc không được trước ngày bắt đầu')] doesn't exist
at org.testng.Assert.fail(Assert.java:97)
at Common.Utilities.assertElementVisible(Utilities.java:585)
at CommonScreen.EnterPlanScreen.checkMultiDate(EnterPlanScreen.java:145)
at TestSuite.EnterPlan.MODULE2_11(EnterPlan.java:86)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:687)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:140)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at org.testng.TestRunner.privateRun(TestRunner.java:724)
at org.testng.TestRunner.run(TestRunner.java:574)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)
at org.testng.SuiteRunner.run(SuiteRunner.java:302)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1142)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1064)
at org.testng.TestNG.runSuites(TestNG.java:994)
at org.testng.TestNG.run(TestNG.java:962)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_11 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:43 ICT" name="tearDownMethod" finished-at="2022-09-20T01:56:43 ICT" duration-ms="239" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_11 status=FAILURE method=EnterPlan.MODULE2_11()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_11() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:43 ICT" name="afterMethod" finished-at="2022-09-20T01:56:43 ICT" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_11 status=FAILURE method=EnterPlan.MODULE2_11()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_11() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:43 ICT" name="setUpMethod" finished-at="2022-09-20T01:56:48 ICT" duration-ms="5015" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_12() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_12()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:56:48 ICT" name="MODULE2_12" finished-at="2022-09-20T01:57:00 ICT" duration-ms="11599" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_12 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:00 ICT" name="tearDownMethod" finished-at="2022-09-20T01:57:00 ICT" duration-ms="76" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_12 status=SUCCESS method=EnterPlan.MODULE2_12()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_12() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:00 ICT" name="afterMethod" finished-at="2022-09-20T01:57:00 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_12 status=SUCCESS method=EnterPlan.MODULE2_12()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_12() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:00 ICT" name="setUpMethod" finished-at="2022-09-20T01:57:05 ICT" duration-ms="5047" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_13() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_13()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:05 ICT" name="MODULE2_13" finished-at="2022-09-20T01:57:18 ICT" duration-ms="12808" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_13 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:18 ICT" name="tearDownMethod" finished-at="2022-09-20T01:57:18 ICT" duration-ms="79" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_13 status=SUCCESS method=EnterPlan.MODULE2_13()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_13() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:18 ICT" name="afterMethod" finished-at="2022-09-20T01:57:18 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_13 status=SUCCESS method=EnterPlan.MODULE2_13()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_13() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:18 ICT" name="setUpMethod" finished-at="2022-09-20T01:57:23 ICT" duration-ms="4961" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_14() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_14()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:23 ICT" name="MODULE2_14" finished-at="2022-09-20T01:57:24 ICT" duration-ms="624" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Element with locator By.xpath: //p[contains(text(),'Ngày không được phép nhỏ hơn giá trị nhỏ nhất')] doesn't exist]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Element with locator By.xpath: //p[contains(text(),'Ngày không được phép nhỏ hơn giá trị nhỏ nhất')] doesn't exist
at org.testng.Assert.fail(Assert.java:97)
at Common.Utilities.assertElementVisible(Utilities.java:585)
at CommonScreen.EnterPlanScreen.checkDate(EnterPlanScreen.java:138)
at TestSuite.EnterPlan.MODULE2_14(EnterPlan.java:101)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:687)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:140)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at org.testng.TestRunner.privateRun(TestRunner.java:724)
at org.testng.TestRunner.run(TestRunner.java:574)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)
at org.testng.SuiteRunner.run(SuiteRunner.java:302)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1142)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1064)
at org.testng.TestNG.runSuites(TestNG.java:994)
at org.testng.TestNG.run(TestNG.java:962)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_14 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:24 ICT" name="tearDownMethod" finished-at="2022-09-20T01:57:24 ICT" duration-ms="250" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_14 status=FAILURE method=EnterPlan.MODULE2_14()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_14() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:24 ICT" name="afterMethod" finished-at="2022-09-20T01:57:24 ICT" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_14 status=FAILURE method=EnterPlan.MODULE2_14()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_14() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:24 ICT" name="setUpMethod" finished-at="2022-09-20T01:57:29 ICT" duration-ms="5156" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_15() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_15()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:29 ICT" name="MODULE2_15" finished-at="2022-09-20T01:57:30 ICT" duration-ms="552" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Element with locator By.xpath: //p[contains(text(),'Ngày không được phép lớn hơn giá trị lớn nhất')] doesn't exist]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Element with locator By.xpath: //p[contains(text(),'Ngày không được phép lớn hơn giá trị lớn nhất')] doesn't exist
at org.testng.Assert.fail(Assert.java:97)
at Common.Utilities.assertElementVisible(Utilities.java:585)
at CommonScreen.EnterPlanScreen.checkDate(EnterPlanScreen.java:138)
at TestSuite.EnterPlan.MODULE2_15(EnterPlan.java:106)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:687)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:140)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at org.testng.TestRunner.privateRun(TestRunner.java:724)
at org.testng.TestRunner.run(TestRunner.java:574)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)
at org.testng.SuiteRunner.run(SuiteRunner.java:302)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1142)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1064)
at org.testng.TestNG.runSuites(TestNG.java:994)
at org.testng.TestNG.run(TestNG.java:962)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_15 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:30 ICT" name="tearDownMethod" finished-at="2022-09-20T01:57:30 ICT" duration-ms="243" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_15 status=FAILURE method=EnterPlan.MODULE2_15()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_15() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:30 ICT" name="afterMethod" finished-at="2022-09-20T01:57:30 ICT" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_15 status=FAILURE method=EnterPlan.MODULE2_15()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_15() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:30 ICT" name="setUpMethod" finished-at="2022-09-20T01:57:35 ICT" duration-ms="5142" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_16() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_16()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:35 ICT" name="MODULE2_16" finished-at="2022-09-20T01:57:56 ICT" duration-ms="20856" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[By.xpath: //div[@class='Toastify']//div//div//divis not visible did not expect to find [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: By.xpath: //div[@class='Toastify']//div//div//divis not visible did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at Common.Utilities.assertTextValueVisible(Utilities.java:703)
at CommonScreen.EnterPlanScreen.savePlan(EnterPlanScreen.java:111)
at TestSuite.EnterPlan.MODULE2_16(EnterPlan.java:111)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:687)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:140)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at org.testng.TestRunner.privateRun(TestRunner.java:724)
at org.testng.TestRunner.run(TestRunner.java:574)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)
at org.testng.SuiteRunner.run(SuiteRunner.java:302)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1142)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1064)
at org.testng.TestNG.runSuites(TestNG.java:994)
at org.testng.TestNG.run(TestNG.java:962)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_16 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:56 ICT" name="tearDownMethod" finished-at="2022-09-20T01:57:56 ICT" duration-ms="211" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_16 status=FAILURE method=EnterPlan.MODULE2_16()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_16() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:56 ICT" name="afterMethod" finished-at="2022-09-20T01:57:56 ICT" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_16 status=FAILURE method=EnterPlan.MODULE2_16()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_16() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:57:56 ICT" name="setUpMethod" finished-at="2022-09-20T01:58:01 ICT" duration-ms="4989" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_17() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_17()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:58:01 ICT" name="MODULE2_17" finished-at="2022-09-20T01:58:02 ICT" duration-ms="1169" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Element with locator By.xpath: //p[contains(text(),'Ngày kết thúc không được trước ngày bắt đầu')] doesn't exist]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Element with locator By.xpath: //p[contains(text(),'Ngày kết thúc không được trước ngày bắt đầu')] doesn't exist
at org.testng.Assert.fail(Assert.java:97)
at Common.Utilities.assertElementVisible(Utilities.java:585)
at CommonScreen.EnterPlanScreen.checkMultiDate(EnterPlanScreen.java:145)
at TestSuite.EnterPlan.MODULE2_17(EnterPlan.java:116)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:687)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:140)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at org.testng.TestRunner.privateRun(TestRunner.java:724)
at org.testng.TestRunner.run(TestRunner.java:574)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)
at org.testng.SuiteRunner.run(SuiteRunner.java:302)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1142)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1064)
at org.testng.TestNG.runSuites(TestNG.java:994)
at org.testng.TestNG.run(TestNG.java:962)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_17 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:58:02 ICT" name="tearDownMethod" finished-at="2022-09-20T01:58:02 ICT" duration-ms="215" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_17 status=FAILURE method=EnterPlan.MODULE2_17()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_17() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:58:02 ICT" name="afterMethod" finished-at="2022-09-20T01:58:02 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_17 status=FAILURE method=EnterPlan.MODULE2_17()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_17() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:58:02 ICT" name="setUpMethod" finished-at="2022-09-20T01:58:08 ICT" duration-ms="5228" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_18() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_18()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:58:08 ICT" name="MODULE2_18" finished-at="2022-09-20T01:58:20 ICT" duration-ms="12496" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_18 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:58:20 ICT" name="tearDownMethod" finished-at="2022-09-20T01:58:20 ICT" duration-ms="79" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_18 status=SUCCESS method=EnterPlan.MODULE2_18()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_18() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:58:20 ICT" name="afterMethod" finished-at="2022-09-20T01:58:20 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_18 status=SUCCESS method=EnterPlan.MODULE2_18()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_18() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:58:20 ICT" name="setUpMethod" finished-at="2022-09-20T01:58:25 ICT" duration-ms="4978" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_19() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_19()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:58:25 ICT" name="MODULE2_19" finished-at="2022-09-20T01:58:38 ICT" duration-ms="12916" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_19 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:58:38 ICT" name="tearDownMethod" finished-at="2022-09-20T01:58:38 ICT" duration-ms="81" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_19 status=SUCCESS method=EnterPlan.MODULE2_19()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_19() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:58:38 ICT" name="afterMethod" finished-at="2022-09-20T01:58:38 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_19 status=SUCCESS method=EnterPlan.MODULE2_19()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_19() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:58:38 ICT" name="setUpMethod" finished-at="2022-09-20T01:58:43 ICT" duration-ms="5081" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_20() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_20()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:58:43 ICT" name="MODULE2_20" finished-at="2022-09-20T01:58:52 ICT" duration-ms="8593" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_20 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:58:52 ICT" name="tearDownMethod" finished-at="2022-09-20T01:58:52 ICT" duration-ms="79" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_20 status=SUCCESS method=EnterPlan.MODULE2_20()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_20() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:58:52 ICT" name="afterMethod" finished-at="2022-09-20T01:58:52 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_20 status=SUCCESS method=EnterPlan.MODULE2_20()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_20() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:58:52 ICT" name="setUpMethod" finished-at="2022-09-20T01:58:57 ICT" duration-ms="5001" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_21() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_21()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:58:57 ICT" name="MODULE2_21" finished-at="2022-09-20T01:59:09 ICT" duration-ms="11655" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_21 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:59:09 ICT" name="tearDownMethod" finished-at="2022-09-20T01:59:09 ICT" duration-ms="91" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_21 status=SUCCESS method=EnterPlan.MODULE2_21()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_21() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:59:09 ICT" name="setUpMethod" finished-at="2022-09-20T01:59:15 ICT" duration-ms="5803" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_22() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:59:09 ICT" name="afterMethod" finished-at="2022-09-20T01:59:09 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_21 status=SUCCESS method=EnterPlan.MODULE2_21()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_21() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method signature="MODULE2_22()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:59:15 ICT" name="MODULE2_22" finished-at="2022-09-20T01:59:24 ICT" duration-ms="9440" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_22 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:59:24 ICT" name="tearDownMethod" finished-at="2022-09-20T01:59:24 ICT" duration-ms="86" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_22 status=SUCCESS method=EnterPlan.MODULE2_22()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_22() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:59:24 ICT" name="setUpMethod" finished-at="2022-09-20T01:59:30 ICT" duration-ms="5730" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_23() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:59:24 ICT" name="afterMethod" finished-at="2022-09-20T01:59:24 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_22 status=SUCCESS method=EnterPlan.MODULE2_22()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_22() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method signature="MODULE2_23()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:59:30 ICT" name="MODULE2_23" finished-at="2022-09-20T01:59:46 ICT" duration-ms="15953" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_23 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:59:46 ICT" name="tearDownMethod" finished-at="2022-09-20T01:59:46 ICT" duration-ms="142" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_23 status=SUCCESS method=EnterPlan.MODULE2_23()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_23() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:59:46 ICT" name="afterMethod" finished-at="2022-09-20T01:59:46 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_23 status=SUCCESS method=EnterPlan.MODULE2_23()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_23() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:59:46 ICT" name="setUpMethod" finished-at="2022-09-20T01:59:52 ICT" duration-ms="5656" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_24() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_24()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T01:59:52 ICT" name="MODULE2_24" finished-at="2022-09-20T02:00:01 ICT" duration-ms="9348" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_24 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:01 ICT" name="tearDownMethod" finished-at="2022-09-20T02:00:01 ICT" duration-ms="78" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_24 status=SUCCESS method=EnterPlan.MODULE2_24()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_24() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:01 ICT" name="afterMethod" finished-at="2022-09-20T02:00:01 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_24 status=SUCCESS method=EnterPlan.MODULE2_24()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_24() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:01 ICT" name="setUpMethod" finished-at="2022-09-20T02:00:06 ICT" duration-ms="5301" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_25() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_25()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:06 ICT" name="MODULE2_25" finished-at="2022-09-20T02:00:18 ICT" duration-ms="12082" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_25 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:18 ICT" name="tearDownMethod" finished-at="2022-09-20T02:00:18 ICT" duration-ms="82" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_25 status=SUCCESS method=EnterPlan.MODULE2_25()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_25() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:18 ICT" name="afterMethod" finished-at="2022-09-20T02:00:18 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_25 status=SUCCESS method=EnterPlan.MODULE2_25()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_25() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:18 ICT" name="setUpMethod" finished-at="2022-09-20T02:00:23 ICT" duration-ms="5061" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_26() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_26()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:23 ICT" name="MODULE2_26" finished-at="2022-09-20T02:00:35 ICT" duration-ms="11903" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_26 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:35 ICT" name="tearDownMethod" finished-at="2022-09-20T02:00:35 ICT" duration-ms="81" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_26 status=SUCCESS method=EnterPlan.MODULE2_26()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_26() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:35 ICT" name="afterMethod" finished-at="2022-09-20T02:00:35 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_26 status=SUCCESS method=EnterPlan.MODULE2_26()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_26() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:35 ICT" name="setUpMethod" finished-at="2022-09-20T02:00:41 ICT" duration-ms="5058" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_27() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_27()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:41 ICT" name="MODULE2_27" finished-at="2022-09-20T02:00:41 ICT" duration-ms="572" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Element with locator By.xpath: //p[contains(text(),'Ngày không được phép nhỏ hơn giá trị nhỏ nhất')] doesn't exist]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Element with locator By.xpath: //p[contains(text(),'Ngày không được phép nhỏ hơn giá trị nhỏ nhất')] doesn't exist
at org.testng.Assert.fail(Assert.java:97)
at Common.Utilities.assertElementVisible(Utilities.java:585)
at CommonScreen.EnterPlanScreen.checkDate(EnterPlanScreen.java:138)
at TestSuite.EnterPlan.MODULE2_27(EnterPlan.java:169)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:687)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:140)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at org.testng.TestRunner.privateRun(TestRunner.java:724)
at org.testng.TestRunner.run(TestRunner.java:574)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)
at org.testng.SuiteRunner.run(SuiteRunner.java:302)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1142)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1064)
at org.testng.TestNG.runSuites(TestNG.java:994)
at org.testng.TestNG.run(TestNG.java:962)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_27 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:41 ICT" name="tearDownMethod" finished-at="2022-09-20T02:00:41 ICT" duration-ms="230" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_27 status=FAILURE method=EnterPlan.MODULE2_27()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_27() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:41 ICT" name="afterMethod" finished-at="2022-09-20T02:00:41 ICT" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_27 status=FAILURE method=EnterPlan.MODULE2_27()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_27() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:41 ICT" name="setUpMethod" finished-at="2022-09-20T02:00:46 ICT" duration-ms="5010" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_28() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_28()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:46 ICT" name="MODULE2_28" finished-at="2022-09-20T02:00:47 ICT" duration-ms="573" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Element with locator By.xpath: //p[contains(text(),'Ngày không được phép lớn hơn giá trị lớn nhất')] doesn't exist]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Element with locator By.xpath: //p[contains(text(),'Ngày không được phép lớn hơn giá trị lớn nhất')] doesn't exist
at org.testng.Assert.fail(Assert.java:97)
at Common.Utilities.assertElementVisible(Utilities.java:585)
at CommonScreen.EnterPlanScreen.checkDate(EnterPlanScreen.java:138)
at TestSuite.EnterPlan.MODULE2_28(EnterPlan.java:174)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:687)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:140)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at org.testng.TestRunner.privateRun(TestRunner.java:724)
at org.testng.TestRunner.run(TestRunner.java:574)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)
at org.testng.SuiteRunner.run(SuiteRunner.java:302)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1142)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1064)
at org.testng.TestNG.runSuites(TestNG.java:994)
at org.testng.TestNG.run(TestNG.java:962)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_28 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:47 ICT" name="tearDownMethod" finished-at="2022-09-20T02:00:47 ICT" duration-ms="229" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_28 status=FAILURE method=EnterPlan.MODULE2_28()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_28() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:47 ICT" name="afterMethod" finished-at="2022-09-20T02:00:47 ICT" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_28 status=FAILURE method=EnterPlan.MODULE2_28()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_28() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:47 ICT" name="setUpMethod" finished-at="2022-09-20T02:00:52 ICT" duration-ms="5234" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_29() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_29()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:00:52 ICT" name="MODULE2_29" finished-at="2022-09-20T02:01:04 ICT" duration-ms="11567" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_29 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:04 ICT" name="tearDownMethod" finished-at="2022-09-20T02:01:04 ICT" duration-ms="81" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_29 status=SUCCESS method=EnterPlan.MODULE2_29()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_29() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:04 ICT" name="afterMethod" finished-at="2022-09-20T02:01:04 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_29 status=SUCCESS method=EnterPlan.MODULE2_29()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_29() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:04 ICT" name="setUpMethod" finished-at="2022-09-20T02:01:09 ICT" duration-ms="4977" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_30() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_30()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:09 ICT" name="MODULE2_30" finished-at="2022-09-20T02:01:10 ICT" duration-ms="1058" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Element with locator By.xpath: //p[contains(text(),'Ngày kết thúc không được trước ngày bắt đầu')] doesn't exist]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Element with locator By.xpath: //p[contains(text(),'Ngày kết thúc không được trước ngày bắt đầu')] doesn't exist
at org.testng.Assert.fail(Assert.java:97)
at Common.Utilities.assertElementVisible(Utilities.java:585)
at CommonScreen.EnterPlanScreen.checkMultiDate(EnterPlanScreen.java:145)
at TestSuite.EnterPlan.MODULE2_30(EnterPlan.java:184)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:687)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:140)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at org.testng.TestRunner.privateRun(TestRunner.java:724)
at org.testng.TestRunner.run(TestRunner.java:574)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)
at org.testng.SuiteRunner.run(SuiteRunner.java:302)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1142)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1064)
at org.testng.TestNG.runSuites(TestNG.java:994)
at org.testng.TestNG.run(TestNG.java:962)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_30 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:10 ICT" name="tearDownMethod" finished-at="2022-09-20T02:01:10 ICT" duration-ms="235" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_30 status=FAILURE method=EnterPlan.MODULE2_30()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_30() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:10 ICT" name="afterMethod" finished-at="2022-09-20T02:01:10 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_30 status=FAILURE method=EnterPlan.MODULE2_30()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_30() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:10 ICT" name="setUpMethod" finished-at="2022-09-20T02:01:15 ICT" duration-ms="5063" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_31() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_31()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:15 ICT" name="MODULE2_31" finished-at="2022-09-20T02:01:27 ICT" duration-ms="12067" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_31 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:27 ICT" name="tearDownMethod" finished-at="2022-09-20T02:01:28 ICT" duration-ms="79" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_31 status=SUCCESS method=EnterPlan.MODULE2_31()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_31() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:28 ICT" name="afterMethod" finished-at="2022-09-20T02:01:28 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_31 status=SUCCESS method=EnterPlan.MODULE2_31()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_31() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:28 ICT" name="setUpMethod" finished-at="2022-09-20T02:01:33 ICT" duration-ms="5101" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_32() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_32()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:33 ICT" name="MODULE2_32" finished-at="2022-09-20T02:01:45 ICT" duration-ms="12811" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_32 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:45 ICT" name="tearDownMethod" finished-at="2022-09-20T02:01:46 ICT" duration-ms="84" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_32 status=SUCCESS method=EnterPlan.MODULE2_32()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_32() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:46 ICT" name="afterMethod" finished-at="2022-09-20T02:01:46 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_32 status=SUCCESS method=EnterPlan.MODULE2_32()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_32() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:46 ICT" name="setUpMethod" finished-at="2022-09-20T02:01:51 ICT" duration-ms="5040" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_33() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_33()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:51 ICT" name="MODULE2_33" finished-at="2022-09-20T02:01:51 ICT" duration-ms="600" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Element with locator By.xpath: //p[contains(text(),'Ngày không được phép nhỏ hơn giá trị nhỏ nhất')] doesn't exist]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Element with locator By.xpath: //p[contains(text(),'Ngày không được phép nhỏ hơn giá trị nhỏ nhất')] doesn't exist
at org.testng.Assert.fail(Assert.java:97)
at Common.Utilities.assertElementVisible(Utilities.java:585)
at CommonScreen.EnterPlanScreen.checkDate(EnterPlanScreen.java:138)
at TestSuite.EnterPlan.MODULE2_33(EnterPlan.java:199)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:687)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:140)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at org.testng.TestRunner.privateRun(TestRunner.java:724)
at org.testng.TestRunner.run(TestRunner.java:574)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)
at org.testng.SuiteRunner.run(SuiteRunner.java:302)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1142)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1064)
at org.testng.TestNG.runSuites(TestNG.java:994)
at org.testng.TestNG.run(TestNG.java:962)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_33 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:51 ICT" name="tearDownMethod" finished-at="2022-09-20T02:01:51 ICT" duration-ms="247" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_33 status=FAILURE method=EnterPlan.MODULE2_33()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_33() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:51 ICT" name="afterMethod" finished-at="2022-09-20T02:01:51 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_33 status=FAILURE method=EnterPlan.MODULE2_33()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_33() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:51 ICT" name="setUpMethod" finished-at="2022-09-20T02:01:57 ICT" duration-ms="5145" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_34() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_34()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:57 ICT" name="MODULE2_34" finished-at="2022-09-20T02:01:57 ICT" duration-ms="568" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Element with locator By.xpath: //p[contains(text(),'Ngày không được phép lớn hơn giá trị lớn nhất')] doesn't exist]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Element with locator By.xpath: //p[contains(text(),'Ngày không được phép lớn hơn giá trị lớn nhất')] doesn't exist
at org.testng.Assert.fail(Assert.java:97)
at Common.Utilities.assertElementVisible(Utilities.java:585)
at CommonScreen.EnterPlanScreen.checkDate(EnterPlanScreen.java:138)
at TestSuite.EnterPlan.MODULE2_34(EnterPlan.java:204)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:687)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:140)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at org.testng.TestRunner.privateRun(TestRunner.java:724)
at org.testng.TestRunner.run(TestRunner.java:574)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)
at org.testng.SuiteRunner.run(SuiteRunner.java:302)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1142)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1064)
at org.testng.TestNG.runSuites(TestNG.java:994)
at org.testng.TestNG.run(TestNG.java:962)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_34 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:57 ICT" name="tearDownMethod" finished-at="2022-09-20T02:01:57 ICT" duration-ms="243" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_34 status=FAILURE method=EnterPlan.MODULE2_34()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_34() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:57 ICT" name="afterMethod" finished-at="2022-09-20T02:01:57 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_34 status=FAILURE method=EnterPlan.MODULE2_34()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_34() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:01:57 ICT" name="setUpMethod" finished-at="2022-09-20T02:02:03 ICT" duration-ms="5133" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_35() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_35()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:02:03 ICT" name="MODULE2_35" finished-at="2022-09-20T02:02:23 ICT" duration-ms="20932" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[By.xpath: //div[@class='Toastify']//div//div//divis not visible did not expect to find [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: By.xpath: //div[@class='Toastify']//div//div//divis not visible did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:751)
at Common.Utilities.assertTextValueVisible(Utilities.java:703)
at CommonScreen.EnterPlanScreen.draftSavePlan(EnterPlanScreen.java:118)
at TestSuite.EnterPlan.MODULE2_35(EnterPlan.java:209)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:687)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:140)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at org.testng.TestRunner.privateRun(TestRunner.java:724)
at org.testng.TestRunner.run(TestRunner.java:574)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)
at org.testng.SuiteRunner.run(SuiteRunner.java:302)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1142)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1064)
at org.testng.TestNG.runSuites(TestNG.java:994)
at org.testng.TestNG.run(TestNG.java:962)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_35 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:02:23 ICT" name="tearDownMethod" finished-at="2022-09-20T02:02:24 ICT" duration-ms="222" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_35 status=FAILURE method=EnterPlan.MODULE2_35()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_35() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:02:24 ICT" name="afterMethod" finished-at="2022-09-20T02:02:24 ICT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_35 status=FAILURE method=EnterPlan.MODULE2_35()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_35() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:02:24 ICT" name="setUpMethod" finished-at="2022-09-20T02:02:29 ICT" duration-ms="5182" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_36() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_36()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:02:29 ICT" name="MODULE2_36" finished-at="2022-09-20T02:02:30 ICT" duration-ms="1258" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Element with locator By.xpath: //p[contains(text(),'Ngày kết thúc không được trước ngày bắt đầu')] doesn't exist]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Element with locator By.xpath: //p[contains(text(),'Ngày kết thúc không được trước ngày bắt đầu')] doesn't exist
at org.testng.Assert.fail(Assert.java:97)
at Common.Utilities.assertElementVisible(Utilities.java:585)
at CommonScreen.EnterPlanScreen.checkMultiDate(EnterPlanScreen.java:145)
at TestSuite.EnterPlan.MODULE2_36(EnterPlan.java:214)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:687)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:140)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at org.testng.TestRunner.privateRun(TestRunner.java:724)
at org.testng.TestRunner.run(TestRunner.java:574)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)
at org.testng.SuiteRunner.run(SuiteRunner.java:302)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1142)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1064)
at org.testng.TestNG.runSuites(TestNG.java:994)
at org.testng.TestNG.run(TestNG.java:962)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_36 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:02:30 ICT" name="tearDownMethod" finished-at="2022-09-20T02:02:30 ICT" duration-ms="261" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_36 status=FAILURE method=EnterPlan.MODULE2_36()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_36() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:02:30 ICT" name="afterMethod" finished-at="2022-09-20T02:02:30 ICT" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_36 status=FAILURE method=EnterPlan.MODULE2_36()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_36() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:02:30 ICT" name="setUpMethod" finished-at="2022-09-20T02:02:35 ICT" duration-ms="5038" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_37() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_37()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:02:35 ICT" name="MODULE2_37" finished-at="2022-09-20T02:02:48 ICT" duration-ms="12399" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_37 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:02:48 ICT" name="tearDownMethod" finished-at="2022-09-20T02:02:48 ICT" duration-ms="78" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_37 status=SUCCESS method=EnterPlan.MODULE2_37()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_37() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:02:48 ICT" name="setUpMethod" finished-at="2022-09-20T02:02:53 ICT" duration-ms="5022" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_38() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:02:48 ICT" name="afterMethod" finished-at="2022-09-20T02:02:48 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_37 status=SUCCESS method=EnterPlan.MODULE2_37()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_37() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method signature="MODULE2_38()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:02:53 ICT" name="MODULE2_38" finished-at="2022-09-20T02:03:06 ICT" duration-ms="12817" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_38 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:03:06 ICT" name="tearDownMethod" finished-at="2022-09-20T02:03:06 ICT" duration-ms="78" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_38 status=SUCCESS method=EnterPlan.MODULE2_38()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_38() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:03:06 ICT" name="afterMethod" finished-at="2022-09-20T02:03:06 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_38 status=SUCCESS method=EnterPlan.MODULE2_38()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_38() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:03:06 ICT" name="setUpMethod" finished-at="2022-09-20T02:03:11 ICT" duration-ms="4958" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_39() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_39()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:03:11 ICT" name="MODULE2_39" finished-at="2022-09-20T02:03:13 ICT" duration-ms="2263" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_39 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:03:13 ICT" name="tearDownMethod" finished-at="2022-09-20T02:03:13 ICT" duration-ms="73" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_39 status=SUCCESS method=EnterPlan.MODULE2_39()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_39() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:03:13 ICT" name="afterMethod" finished-at="2022-09-20T02:03:13 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_39 status=SUCCESS method=EnterPlan.MODULE2_39()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_39() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:03:13 ICT" name="setUpMethod" finished-at="2022-09-20T02:03:18 ICT" duration-ms="5266" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_40() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="MODULE2_40()[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:03:18 ICT" name="MODULE2_40" finished-at="2022-09-20T02:03:21 ICT" duration-ms="2247" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MODULE2_40 -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:03:21 ICT" name="tearDownMethod" finished-at="2022-09-20T02:03:21 ICT" duration-ms="74" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_40 status=SUCCESS method=EnterPlan.MODULE2_40()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_40() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:TestSuite.EnterPlan@75329a49]" started-at="2022-09-20T02:03:21 ICT" name="afterMethod" finished-at="2022-09-20T02:03:21 ICT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=MODULE2_40 status=SUCCESS method=EnterPlan.MODULE2_40()[pri:0, instance:TestSuite.EnterPlan@75329a49] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestSuite.EnterPlan.MODULE2_40() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- TestSuite.EnterPlan -->
    </test> <!-- Test1 -->
  </suite> <!-- Suite -->
</testng-results>
